experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -2316,6 +2329,80 @@ def _validate_primary_secondary_outcomes(self, data):
 
         return data
 
+    def _validate_firefox_labs(self, data):
+        if not data.get("is_firefox_labs_opt_in"):
+            return data
+
+        min_version = NimbusExperiment.Version.parse(data.get("firefox_min_version"))
+        required_min_version = NimbusExperiment.FIREFOX_LABS_MIN_VERSION.get(
+            self.instance.application
+        )
+
+        if required_min_version is None:
+            raise serializers.ValidationError(
+                {
+                    "is_firefox_labs_opt_in": NimbusExperiment.ERROR_FIREFOX_LABS_UNSUPPORTED_APPLICATION,
+                }
+            )
+        else:
+            required_min_version = NimbusExperiment.Version.parse(required_min_version)
+            if min_version < required_min_version:
+                raise serializers.ValidationError(
+                    {
+                        "firefox_min_version": NimbusExperiment.ERROR_FIREFOX_LABS_MIN_VERSION.format(
+                            version=required_min_version
+                        ),
+                    }
+                )
+
+        if not data.get("is_rollout"):
+            raise serializers.ValidationError(
+                {"is_rollout": NimbusExperiment.ERROR_FIREFOX_LABS_ROLLOUT_REQUIRED}
+            )
+
+        errors = {
+            field: [NimbusExperiment.ERROR_FIREFOX_LABS_REQUIRED_FIELD]
+            for field in (
+                "firefox_labs_title",
+                "firefox_labs_description",
+                "firefox_labs_group",
+            )
+            if not len((data.get(field) or "").strip())
+        }
+
+        if description_links := (
+            data.get("firefox_labs_description_links") or ""
```

Comment:
dict.get(key, fallback) you can use the fallback instead of `or`ing, not a huge difference